packages/angular/build/src/builders/dev-server/vite-server.ts

```diff
@@ -117,6 +117,10 @@ export async function* serveWithVite(
     autoCsp: false,
   };
 
+  // Disable JSON build stats.
+  // These are not accessible with the dev server and can cause HMR fallbacks.
+  browserOptions.statsJson = false;
```

Comment:
Question: Is it worth logging that we're ignoring `statsJson`? I could see some devs getting a bit confused that they enable it but nothing happens.